rules/no-native.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
     /**
      * Checks for and reports reassigned constants
      *
-     * @param {Scope} scope - an escope Scope object
+     * @param {Scope} scope - an scope Scope object
```

Comment:
I think `escope` is an npm package. So I don't think it's a typo.
https://www.npmjs.com/package/escope

```suggestion
     * @param {Scope} scope - an escope Scope object
```

Reply:
Maybe that was the package some time ago, looking at the blame, that comment was back from ESLint 3 days. It seems like it's an internal definition https://github.com/nschonni/eslint/blob/757cd9d13160f045fbd0a30100eb0913e8229bfa/docs/src/developer-guide/working-with-rules.md#L164 so I'm not sure this comment is very helpful anymore

Reply:
ESLint currently uses the `eslint-scope` package. The `eslint-scope` is a fork of `escope`.
https://github.com/eslint/eslint-scope

If you want to fix the JSDoc for current state, I think the correct way to do it is:
```suggestion
     * @param {Scope} scope - an eslint-scope Scope object
```